config.plist/security.md

```diff
@@ -0,0 +1,42 @@
+# Apple Secure Boot
+
+These settings in your config.plist can restrict which macOS versions OpenCore will boot. You will want to check these really quick before booting your USB.
+
+## Misc
+
+### Security -> SecureBootModel
+
+OpenCore by default has [Apple Secure Boot](https://dortania.github.io/OpenCore-Post-Install/universal/security/applesecureboot.html#what-is-apple-secure-boot) enabled.
+This enables security features such as verification of macOS' `boot.efi`, though also has the side effect of restricting which macOS versions OpenCore will boot.
+
+* Big Sur and Above (11.0+): The recommended value is `Default`
+* High Sierra-Catalina (10.13-10.15):
+  * If your model is not listed below, set to `Disabled`
+  * If running Nvidia Web Drivers, set to `Disabled`
+  * If your model is listed, compare the minimum version with the version your installing. Disable if your installer is below the minimum version listed for your SMBIOS.
+* Sierra and Below (10.4-10.12): Set to `Disabled`
```

Comment:
Sierra and below aren't affected by SBM

> Operating systems released before Apple Secure Boot was released (e.g. macOS 10.12 or earlier), will still boot until UEFI Secure Boot is enabled. This is so because Apple Secure Boot treats these as incompatible and they are then handled by the firmware (as Microsoft Windows is).